app/models/tweet.rb

```diff
@@ -73,7 +73,7 @@ def make_tweet_from_api_object(tweeted)
       tweet = Tweet.where(twitter_id_code: tweeted.attrs[:id_str]).first_or_initialize
       tweet.twitter_uid = tweeted.user.id.to_s
       tweet.twitter_username = tweeted.user.screen_name.downcase
-      tweet.user_id = User.find_by(twitter_username: tweeted.user.screen_name).try(:id)
+      tweet.user_id = User.find_by(twitter_username: tweeted.user.screen_name).&(:id)
```

Comment:
actually, that would raise `NoMethodError: undefined method `&' for #<User:....>`

It should be the following
```suggestion
      tweet.user_id = User.find_by(twitter_username: tweeted.user.screen_name)&.id
```

Reply:
This tells me two things:
1) I am not thorough enough.
2) Automated tests need some love here.

😅 Thanks.

Reply:
I nearly missed it too. This section is definitely missing test.